### ISSUE TITLE
Prometheus v3.0.0 Issue: non-compliant scrape target sending blank Content-Type

### ISSUE THREAD

**Issue by onedr0p** [slskd/slskd#1252]
I recently updated from prometheus v2 to v3 and noticed this issue. While the work around is mentioned in the error it would be great to be fixed up here.

```
Error scraping target: non-compliant scrape target sending blank Content-Type and no fallback_scrape_protocol specified for target
```

Ref: https://prometheus.io/docs/prometheus/3.0/migration/#scrape-protocols

The only response headers on slskd I see are:

```
❯ curl -sS -D - https://slskd/metrics -o /dev/null
HTTP/2 200
date: Sun, 08 Dec 2024 22:13:00 GMT
strict-transport-security: max-age=31449600; includeSubDomains
```

Here's a comparison to node_exporter response headers:

```
❯ curl -sS -D - http://node-exporter/metrics -o /dev/null
HTTP/1.1 200 OK
Content-Type: text/plain; version=0.0.4; charset=utf-8; escaping=values
Date: Sun, 08 Dec 2024 22:14:15 GMT
Transfer-Encoding: chunked
```

**Comment by onedr0p** [comment]
Interesting, I do see the content-type on this endpoint.

```
❯ curl -sS -D - https://slskd/api/v0/metrics -o /dev/null
HTTP/2 200
date: Sun, 08 Dec 2024 22:17:18 GMT
content-type: text/plain
content-length: 119726
api-supported-versions: 0
strict-transport-security: max-age=31449600; includeSubDomains
```

**Comment by jpdillingham** [comment]
This should have been fixed in #1245 but prior to a few minutes ago it wasn't released.  The [0.22.0](https://github.com/slskd/slskd/releases/tag/0.22.0) build should be ready about 20 minutes from this post, let me know if it works.

**Comment by onedr0p** [comment]
Looks good, thanks 👍
